cmd/operator-sdk/printdeps/cmd.go

```diff
@@ -30,15 +33,15 @@ func NewCmd() *cobra.Command {
 		Short: "Print Golang packages and versions required to run the operator",
 		Long: `The operator-sdk print-deps command prints all Golang packages and versions expected
 by this version of the Operator SDK. Versions for these packages should match
-those in an operators' Gopkg.toml file.
+those in an operators' go.mod or Gopkg.toml file.
 
 print-deps prints in columnar format by default. Use the --as-file flag to
-print in Gopkg.toml file format.
+print in go.mod or Gopkg.toml file format.
```

Comment:
Eh its an anachronism from when terminals were only 80 characters wide. Its not necessary with new terminals because of line wrapping. I can join the lines.

Reply:
Yes, agreed we don't need to stick with 80, or at least be consistent throughout. I think most CLI tools moved away from that.

Reply:
Actually I noticed that most SDK commands have 80-character width help texts, so I'm going to leave this as-is for now. We can address this in a separate PR if we feel the need.